template-compose/app/build.gradle.kts

```diff
@@ -124,7 +124,6 @@ dependencies {
 
     implementation("androidx.activity:activity-ktx:${Versions.ANDROIDX_ACTIVITY_KTX_VERSION}")
     implementation("androidx.appcompat:appcompat:${Versions.ANDROIDX_SUPPORT_VERSION}")
```

Comment:
```suggestion
    
```

Reply:
Shall we also change the extension from `AppCompatActivity` to `ComponentActivity` in `BaseActivity` according to [this thread](https://nimble-co.slack.com/archives/G41UCTGHL/p1669019803168809)? Do we even need `BaseActivity`? @Tuubz

Reply:
I think we should keep "big" changes for separate pull issues (same as switching navigation component tp Kotlin for example).

Would you mind creating an issue for this? 🙏

Reply:
@kaungkhantsoe Thank you for spotting out 👍🏼 updated